db/database.go

```diff
@@ -81,49 +84,57 @@ func (d *SqlxDatabase) performAndReport(fields logging.Fields, fn func() (interf
 	return ret, err
 }
 
-func (d *SqlxDatabase) Get(dest interface{}, query string, args ...interface{}) error {
+func (d *PgxDatabase) GetStruct(dest interface{}, query string, args ...interface{}) error {
```

Comment:
The only issue that I have is that all the other methods behave like xxxStruct and this one got the explicit name.
Looks like Get should get the explicit name in this case.

Reply:
Changing to `Get` and `GetPrimitive` (it's not really a primitive, Go doesn't have those, but it sort-of is, and "scalar" would also be inappropriate).